src/intl/en/page-staking.json

```diff
@@ -32,10 +32,11 @@
   "page-staking-more-sharding": "More on sharding",
   "page-staking-pool": "Stake in a pool",
   "page-staking-pool-desc": "If you have less than 32 ETH, you will be able to add a smaller stake to staking pools. Some companies can do it all on your behalf so you won't have to worry about staying online. Here are some companies to check out.",
+  "page-staking-rocket-pool": "Rocket Pool allows users with only 16 ETH to run a node on the Ethereum network by pairing the node operator's ETH with 16 more from the protocol's deposit pool. If you'd like to participate in the protocol without running a node, you can swap ETH for their liquid token rETH with as little as 0.01 ETH. This is the only truly decentralized staking pool.",
```

Comment:
```suggestion
  "page-staking-rocket-pool": "Using Rocket Pool, you can run a staking node starting from 16 ETH. The remaining ETH gets assigned from the Rocket Pool protocol, using ETH users have deposited to the pool. If you'd like to participate in the protocol without running a node, you can swap ETH for their liquid token rETH with as little as 0.01 ETH. This is the only truly decentralized staking pool.",
```

Reply:
I feel like mentioning staking as a user (vs a node operator) should be the first thing to mention.